substrate/frame/safe-mode/src/lib.rs

```diff
@@ -15,6 +15,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Safe Mode
+//!
+//! Trigger for stopping all extrinsics outside of a specific whitelist.
+//!
+//! ## WARNING
+//!
+//! NOT YET AUDITED. DO NOT USE IN PRODUCTION.
+//!
+//! ## Pallet API
+//!
+//! See the [`pallet`] module for more information about the interfaces this pallet exposes,
+//! including its configuration trait, dispatchables, storage items, events, and errors.
+//!
+//! ## Overview
+//!
+//! Safe mode is entered via two paths (deposit or forced) until a set block number.
+//! The mode is exited when the block number is reached or a call to one of the exit extrinsics is
+//! made. A `WhitelistedCalls` configuration contains all calls that can be executed while in
+//! safe mode.
```

Comment:
Line length.
```suggestion
//! Safe mode is entered via two paths (deposit or forced) until a set block number. The mode is
//! exited when the block number is reached or a call to one of the exit extrinsics is made. A
//! `WhitelistedCalls` configuration contains all calls that can be executed while in safe mode.
```

Reply:
@joepetrowski (Using this thread for all the comments of this type). Here's the pattern I followed that ended up with the specific line length:

1. Each sentence should be on its own line. Found this is best for git to highlight only the changes and markdown and as far as I have seen cargo docs render it the same.
2. Run `cargo fmt` from root which is what formats the line length.

Perhaps I am missing the correct fmt command?

Reply:
The line length in our project is default to exactly 100, the contributing docs only mention "not longer" https://github.com/paritytech/polkadot-sdk/blob/master/docs/STYLE_GUIDE.md#formatting but its actually an "exact 100".  
So its enough to write the text and then call fmt to have the correct breaks.

Reply:
(1) makes sense for git, but when reading unrendered I think it's harder to read. (2) I don't think cargo does this, but am maybe wrong?

Reply:
I can confirm that cargo does do it, as it makes the needed changes.

Happy to go either way on the plain text. I find it easier to read, but written a lot of markdown that way so it might just be that I am used to it.